capsules/diffd

```diff
@@ -0,0 +1,1164 @@
+diff --git a/boards/components/src/crc.rs b/boards/components/src/crc.rs
```

Comment:
I assume this wasn't intentional.

Reply:
Lesson learned: keep your repository clean. Thanks @bradjc.